data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql

```diff
@@ -0,0 +1,10 @@
+ -- Wrathscale Sorceress smart ai
+SET @ENTRY := 17336;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryOrGuid` = @ENTRY;
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 0, 0, 0, 0, 0, 100, 0, 3500, 12000, 3500, 12000, 11, 9672, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Every 3.5 - 12 seconds (IC) - Self: Cast spell Frostbolt (9672) on Victim');
```

Comment:
Same as the other PR comment wise.

name - event - action

`Wrathscale Sorceress - In Combat - Cast 'Frostbolt'`